README.md

```diff
@@ -10,7 +10,7 @@ cd DAMNIT
 conda create -n amore python
 
 conda activate amore
-pip install .
+pip install .[gui]
```

Comment:
This'll need quotes:
```suggestion
pip install '.[gui]'
```

(same thing below)

Reply:
Thanks, done :+1: